cross/gitea/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = gitea
+PKG_VERS = 1.15.9
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-src-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/go-gitea/gitea/releases/download/v$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+EXTRACT_PATH = $(WORK_DIR)/$(PKG_DIR)
+
+BUILD_DEPENDS = native/go
+
+HOMEPAGE = https://gitea.io/
+COMMENT  = Gitea is a community managed lightweight code hosting solution written in Go.
+LICENSE  = MIT
+
+CGO_ENABLED = 1
+COMPILE_TARGET = gitea_compile_target
+GO_SRC_DIR = $(WORK_DIR)/$(PKG_DIR)
+GO_BIN_DIR = $(GO_SRC_DIR)/$(PKG_NAME)
+include ../../mk/spksrc.cross-go.mk
+
+SYNOPKG_PKGVAR = /var/packages/$(PKG_NAME)/var
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+SYNOPKG_PKGVAR = /var/packages/$(PKG_NAME)/target/var
+endif
```

Comment:
`SYNOPKG_PKGVAR = /var/packages/$(PKG_NAME)/var`  can be used on all DSM versions, as  the generic installers for dsm5 and dsm6 link the target/var folder to this
in postinst with:
```
    # Link for DSM7 forward compatibility of var location
    $LN "${SYNOPKG_PKGDEST}/var" "/var/packages/${SYNOPKG_PKGNAME}/var" 2>&1 | install_log
```
```suggestion
```

Reply:
Thanks, I wasn't aware of this nice change :+1: